components/new-contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -58,6 +62,8 @@ class ContributionFlowStepContainer extends React.Component {
       return this.props.mainState.stepDetails?.interval
         ? this.props.intl.formatMessage(this.headerMessages[`profile.guest.recurrent`])
         : this.props.intl.formatMessage(this.headerMessages[`profile.guest`]);
+    } else if (step === 'payment' && this.props.mainState.stepProfile.blocked) {
```

Comment:
`blocked` still there

Reply:
Could we maybe have props validation to catch that?

Reply:
In this component?

Reply:
PropType validation with `PropTypes.shape` could have helped catch that `blocked` is not defined anymore.

Reply:
But `blocked` would have been null if there weren't defined categories as well